go.mod

```diff
@@ -332,22 +332,21 @@ replace (
 	github.com/liggitt/tabwriter => github.com/liggitt/tabwriter v0.0.0-20181228230101-89fcab3d43de
 	github.com/lithammer/dedent => github.com/lithammer/dedent v1.1.0
 	github.com/lpabon/godbc => github.com/lpabon/godbc v0.1.1
+	github.com/lyft/protoc-gen-star => github.com/lyft/protoc-gen-star v0.5.3
```

Comment:
what is pulling this in? another proto generation tool increases likelihood of selecting incompatible transitive protobuf libraries

Reply:
This appears to makes it's way in from [google.golang.org/grpc](https://github.com/grpc/grpc-go/blob/b4c5e24c7373b22c6c38d68379c39ecdc616a968/go.mod#L9) via [github.com/envoyproxy/go-control-plane](https://github.com/envoyproxy/go-control-plane/blob/main/go.mod#L8) via [github.com/envoyproxy/protoc-gen-validate](https://github.com/envoyproxy/protoc-gen-validate/blob/79071f0f8b04188b297a0517a6e55b2d3641ab5a/go.mod#L7).